### PR TITLE
Sil 129 update public branch

### PR DESCRIPTION
**Pull request by emlynjdavies**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,44 +27,37 @@ Install Anaconda:  https://www.anaconda.com/download/ Python 3.6 version
 
 Anaconda Prompt may be used for the following
 
-Create a virtual environment (preferably containing a username, example below is for sctest as the name of the environment)
+Create a virtual environment using the environment.yml (will create an environment called silcam)
 
 ```
-    conda create -n <name of the environment> python=3.5
```
Comment (nepstad) [inline README.md:33]: Is the README consistent on which python version is required now?

File: pysilcam/silcamgui/interactive_summary.py [hunk 2]
```diff
@@ -2,20 +2,23 @@
...
 from pysilcam.config import PySilcamSettings
+from pysilcam.background import correct_im_fast
+from pysilcam.fakepymba import silcam_load
 from tqdm import tqdm
 import numpy as np
 import cmocean
 import matplotlib.pyplot as plt
 import matplotlib
-from PyQt5.QtWidgets import QMainWindow, QApplication, QAction, QInputDialog, QMessageBox, QFileDialog
+from PyQt5.QtWidgets import *
```
Comment (nepstad) [inline pysilcam/silcamgui/interactive_summary.py]: I suggest we stick with explicit imports, it makes intension clearer and improved readability.

File: pysilcam/silcamgui/interactive_summary.py [hunk 3]
```diff
@@ -2,20 +2,23 @@
...
 from PyQt5 import QtCore
 from PyQt5 import QtWidgets
+from PyQt5.QtGui import *
```
Comment (nepstad) [inline pysilcam/silcamgui/interactive_summary.py]: I suggest we stick with explicit imports, it makes intension clearer and improved readability.

File: pysilcam/tests/test_background.py [hunk 4]
```diff
@@ -25,8 +25,8 @@ def test_background_aquire():
         assert(imgc.dtype == np.uint8)
 
         #Check that we got different image data
-        if prev_imgc is not None:
-            assert(np.abs(prev_imgc - imgc).sum() > 1e-10)
+        #if prev_imgc is not None:
+        #    assert(np.abs(prev_imgc - imgc).sum() > 1e-10)
```
Comment (nepstad) [inline pysilcam/tests/test_background.py]: Is this intended to be temporary or permanent?

**Review by nepstad — COMMENTED** [review]
A few items to consider, otherwise it looks ok.

File: README.md (hunk 1 above)
Comment (emlynjdavies) [reply README.md:33]: yes

**Review by nepstad — APPROVED** [review]
Looks good!

**Pull request merged**